packages/nimble-components/src/table/template.ts

```diff
@@ -95,7 +95,7 @@ export const template = html<Table>`
                                     :dataRecord="${(x, c) => c.parent.tableData[x.index]?.record}"
                                     :columns="${(_, c) => c.parent.columns}"
                                     :nestingLevel="${(x, c) => c.parent.tableData[x.index]?.nestingLevel}"
-                                    @click="${async (x, c) => c.parent.onRowClick(x.index)}"
+                                    @click="${(x, c) => c.parent.onRowClick(x.index)}"
                                     @row-selection-toggle="${async (x, c) => c.parent.onRowSelectionToggle(x.index, c.event as CustomEvent<TableRowSelectionToggleEventDetail>)}"
```

Comment:
Maybe all of these `async` event handlers shouldn't be `async`? As I said in my other comment, it doesn't appear that the resolved value is used, though I guess none of these custom events have default actions (that are being prevented).

Reply:
I agree that they should not be async

Reply:
Eh maybe not, makes the template messy. The handlers themselves should probably not be marked async, could handle in a follow-up / create a tech debt to clean-it up

Reply:
I removed `async` from all the handlers on the table.

Reply:
Looks like the merge from main regressed this change.

Reply:
Somehow "format on save" added those `async` keywords! 😲🤬 Running `npm run format` doesn't add them. I'm baffled...